### PR TITLE
AT_004.001| Freestyle project > Disable/Enable the project

### PR DESCRIPTION
**Pull request by ValeriiSarukhanov**
AT_004.001| Freestyle project > Disable/Enable the project
https://trello.com/c/eI3qW0Ue/245-at004001-freestyle-project-disable-enable-the-project

### CODE REVIEW

**Review by VitaliiArtemenko — APPROVED** [review]
LGTM

File: src/test/java/CreateFreestyleProjectWithoutDescription.java [hunk 1]
```diff
@@ -48,10 +54,20 @@ public void testUserCanConfigureNewProject() {
...
         Assert.assertEquals(alert, "alert to be present");
         Assert.assertEquals(description, "Add description");
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testUserEnableDisableProject() {
+        startToCreateNewFreestyleProject();
+        submitButton();
```
Comment (EvgeniyaPiskunova) [inline src/test/java/CreateFreestyleProjectWithoutDescription.java:65]: it looks weird submitting a form 3x times causes different results after every click

File: src/test/java/CreateFreestyleProjectWithoutDescription.java [hunk 2]
```diff
@@ -48,10 +54,20 @@ public void testUserCanConfigureNewProject() {
...
+
+        submitButton();
+        Assert.assertTrue(getDriver().findElement(By.xpath("//form[contains(text(), 'This project is currently disabled')]")).isDisplayed());
+
+        submitButton();
+        Assert.assertTrue(getDriver().findElement(By.xpath("//form[not(contains(text(), 'This project is currently disabled'))]")).isEnabled());
```
Comment (EvgeniyaPiskunova) [inline src/test/java/CreateFreestyleProjectWithoutDescription.java:71]: any page which does not contains this text will return something. It is better to find some existing evidences of success.

**Review by EvgeniyaPiskunova — CHANGES_REQUESTED** [review]

File: src/test/java/CreateFreestyleProjectWithoutDescription.java (hunk 1 above)
Comment (ValeriiSarukhanov) [reply src/test/java/CreateFreestyleProjectWithoutDescription.java:65]: Try run this test.

File: src/test/java/CreateFreestyleProjectWithoutDescription.java (hunk 2 above)
Comment (ValeriiSarukhanov) [reply src/test/java/CreateFreestyleProjectWithoutDescription.java:71]: Sure, I'll put another locator, if you'll approve it.

File: src/test/java/CreateFreestyleProjectWithoutDescription.java (hunk 1 above)
Comment (VitaliiArtemenko) [reply src/test/java/CreateFreestyleProjectWithoutDescription.java:65]: It's a common cause to have the same button with the same selector but on different pages. So, he's going through pages by clicking only one button. His expected result is on the third page. That's why he should click this button three times.

File: src/test/java/CreateFreestyleProjectWithoutDescription.java (hunk 2 above)
Comment (EvgeniyaPiskunova) [reply src/test/java/CreateFreestyleProjectWithoutDescription.java]: please make changes in this PR

File: src/test/java/CreateFreestyleProjectWithoutDescription.java (hunk 1 above)
Comment (EvgeniyaPiskunova) [reply src/test/java/CreateFreestyleProjectWithoutDescription.java:65]: thank you for the clarification

**Comment by RuslanG13** [comment]
[AT_008 | Pipeline > Create Pipeline](https://trello.com/c/ltC3YCJt/253-at008-pipeline-create-pipeline)

**Comment by ValeriiSarukhanov** [comment]
> 

What is the question?

File: src/test/java/CreateFreestyleProjectWithoutDescription.java (hunk 2 above)
Comment (ValeriiSarukhanov) [reply src/test/java/CreateFreestyleProjectWithoutDescription.java]: Done

**Review by EvgeniyaPiskunova — APPROVED** [review]

**Pull request merged**